src/main/java/no/nav/familie/inntektsmelding/forespørsel/tjenester/ForespørselMapper.java

```diff
@@ -0,0 +1,68 @@
+package no.nav.familie.inntektsmelding.forespørsel.tjenester;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import no.nav.familie.inntektsmelding.forespørsel.modell.DelvisFraværsDagForespørselEntitet;
+import no.nav.familie.inntektsmelding.forespørsel.modell.ForespørselEntitet;
+import no.nav.familie.inntektsmelding.forespørsel.modell.FraværsPeriodeForespørselEntitet;
+import no.nav.familie.inntektsmelding.forespørsel.modell.OmsorgspengerForespørselEntitet;
+import no.nav.familie.inntektsmelding.imdialog.modell.PeriodeEntitet;
+import no.nav.familie.inntektsmelding.koder.Ytelsetype;
+import no.nav.familie.inntektsmelding.typer.dto.DelvisFraværsDagDto;
+import no.nav.familie.inntektsmelding.typer.dto.FraværsPeriodeDto;
+import no.nav.familie.inntektsmelding.typer.dto.OmsorgspengerDataDto;
+import no.nav.familie.inntektsmelding.typer.entitet.AktørIdEntitet;
+
+public class ForespørselMapper {
```

Comment:
En privat konstruktør her er ryddig for å hindre bruk av new() (siden alle metodene er statiske)